docs/multicluster/upgrade.md

```diff
@@ -11,11 +11,11 @@ The goal is to support 'graceful' upgrade. Multi-cluster upgrade will not have d
 to data-plane of member clusters, but there can be downtime of processing new configurations
```

Comment:
I feel we should describe how to upgrade from a version prior to 1.13 with manual steps to add ClusterID to ClusterSet, make sure ClusterSet name matches ClusterSet ID, and optionally remove ClusterClaims.

And in user-guide.md, we should mention the requirement for the manual steps and add a link to the upgrade.md section.

Reply:
Done, the doc and comments are updated, and a new section is added in the upgrade.md.